src/app/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,56 @@
+.sign-in-page-background {
+  height: 100%;
+  background: url("../../assets/auth-background.jpg") no-repeat;
+  background-size: cover;
+}
+
+.container {
+  max-width: 100rem;
+  padding: 0 1rem;
+  height: 100%;
+  margin-inline: auto;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.sign-in-form {
+  max-width: 32rem;
+  width: 100%;
+
+  &__title {
+    font-family: 'impact', sans-serif;
+    font-size: 2.625rem;
+    color: #F8BB15;
+    margin-bottom: 2.5rem;
+  }
+
+  @media (max-width: 400px) {
```

Comment:
you can put meida request inside '&__title {' rule

Reply:
i guess you are mistaken. how the selector will look in this case?

Reply:
)))

`&__title { 
   @media (max-width: 400px) { YOUR_STYLES}
}`

Reply:
fixed